### PR TITLE
docs: link custom validation rules & add custom rules for login

### PR DESCRIPTION
**Pull request by datamweb**
Hello friends,

- add link for custom validation rules
- add description custom rules for login

And I want to know, will adding `Validation.login` to this section make our documentation better or not?

### CODE REVIEW

**Comment by kenjis** [comment]
@lonnieezell 
It seems the TOC is generated by a tool.
We should have add the tool in this repository.

**Review by kenjis — APPROVED** [review]

**Comment by kenjis** [comment]
@datamweb Adding `Validation.login` is no problem.

**Review by kenjis — APPROVED** [review]

File: docs/customization.md [hunk 1]
```diff
@@ -139,3 +142,27 @@ If you need a different set of rules for registration, you can specify them in y
         ],
     ];
 ```
+
+### Login
+
+Similar to what was explained about the setting of validation rules for **Registration** section, you can add the rules of the login form in file (**app/Config/Validation.php**) and change the rules.
```
Comment (MGatner) [inline docs/customization.md:148]:
```suggestion
Similar to the process for validation rules in the **Registration** section, you can add rules for the login form to **app/Config/Validation.php** and change the rules.
```

**Review by MGatner — APPROVED** [review]
Looks good! A phrasing suggestion you may take or leave.

**Review by MGatner — APPROVED** [review]

**Pull request merged**